azure/services/publicips/publicips.go

```diff
@@ -75,6 +76,9 @@ func (s *Service) Reconcile(ctx context.Context) error {
 			}
 		}
 
+		outboundIP := net.ParseIP(ip.OutboundIP)
```

Comment:
We should add mapping from api to scope for the new field `outboundIP` => https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/45519b18be2cb7a26034419df99d46c87b410285/azure/scope/cluster.go#L125

Reply:
something like [this](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/57ca45a908419f8ae0ed2a78e8bacc53db25a41f/azure/scope/cluster.go#L734) ?

Reply:
yeah something like that. To verify the expected behavior, I'd suggest creating a cluster using tilt and see if the appropriate resources are created.

Reply:
ok but how to use existing IP, I mean which IPs can I use for testing?